packages/dashboards/addon/templates/components/dashboard-filters.hbs

```diff
@@ -1,16 +1,16 @@
-{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2020, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 <span class="dashboard-filters__expand-button" role="button" {{on "click" (toggle "isCollapsed" this)}}>
   <NaviIcon @icon="sliders" /> Settings<NaviIcon @icon="angle-down" class="dashboard-filters__toggle" />
 </span>
 
 <FilterCollection
   class="dashboard-filters-collection"
   @isCollapsed={{this.isCollapsed}}
-  @onUpdateCollapsed={{set this.isCollapsed _}}
+  @onUpdateCollapsed={{mut this.isCollapsed}}
```

Comment:
```suggestion
  @onUpdateCollapsed={{fn (mut this.isCollapsed})}
```
??

Reply:
Seems to work fine as is, but we can add this so that it is only expected to be called as a function